src/keboola_mcp_server/client.py

```diff
@@ -355,6 +355,27 @@ async def delete(
         return await self.raw_client.delete(endpoint=endpoint)
 
 
+class GlobalSearchResponse(BaseModel):
+    """The SAPI global search response."""
+
+    all: int = Field(description='Total number of found results.')
+    items: list[dict[str, Any]] = Field(
+        description='List of search results containing the items of the GlobalSearchType.'
+    )
+    by_type: dict[str, JsonPrimitive] = Field(description='Search results grouped by type.', alias='byType')
```

Comment:
why `JsonPrimitive`? Please make it more clear in the description what the key is and what the value is.